docs/docker.md

```diff
@@ -1,4 +1,4 @@
-# Join the testnet with Docker
```

Comment:
Could you please update the link in RELEASE-NOTES.md?

Reply:
done in c928c53